Added About Us page and edited navbar to have more space

manipchi opened the pull request:
- still need to add things to about us page

manipchi committed:
Added About Us page and edited navbar
- still need to add things to about us page

sophiemachin reviewed (changes requested):
I think this is basically good. 

Currently the text is tiny and essentially unreadable so I think we should just get rid of the text. What do you think about using `lion_with_text_2.png` for the image? This doesnt have any text

manipchi committed:
- made logo without text transparent for header
- updated the sign up page by adding the mailing list and getting rid of membership form

manipchi commented:
- fixed in new PR
